bot/code_review_bot/revisions.py

```diff
@@ -105,45 +131,58 @@ def __repr__(self):
     def __str__(self):
         return "Phabricator #{} - {}".format(self.diff_id, self.diff_phid)
 
-    @property
-    def url(self):
-        return "https://{}/D{}".format(self.api.hostname, self.id)
-
-    def load_phabricator(self, try_task):
+    @staticmethod
+    def from_try(try_task, phabricator):
         """
         Load identifiers from Phabricator, using the remote task description
         """
+        assert isinstance(phabricator, PhabricatorAPI)
+        assert isinstance(try_task, dict)
```

Comment:
Nit: you could use types in the function definition

Reply:
Done